.github/workflows/bump-version.yml

```
@@ -0,0 +1,80 @@
+        run: |
+          cp ${{ github.workspace }}/oxen-rust/rust-toolchain.toml .
+
+      - name: Setup Rust Toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v6
+
+      - name: Bump version
+        id: bump
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:3:39: Double quote to prevent globbing and word splitting

(shellcheck)